osu.Game.Rulesets.Catch/UI/CatcherArea.cs

```diff
@@ -57,7 +57,7 @@ void runAfterLoaded(Action action)
 
                 // this is required to make this run after the last caught fruit runs UpdateState at least once.
                 // TODO: find a better alternative
-                if (lastPlateableFruit.IsLoaded)
+                if (lastPlateableFruit.LoadState > LoadState.Ready)
```

Comment:
This doesn't do anything different to the `IsLoaded` check, and would prefer the `IsLoaded` check in this case.

Reply:
Hmm, right. Did we change this at some point?

Either way, this wasn't really supposed to fix anything in the end.